.git-blame-ignore-revs

```diff
@@ -0,0 +1,6 @@
+# A common convention for ignoring commits with automated formatting.
```

Comment:
```suggestion
# File that will be passed as argument to `git blame --ignore-revs-file`
# Name `.git-blame-ignore-revs is a convention.
```

Reply:
Right - it is not automatic, I see that it could be intepreted that way.  I will edit the suggestion again.